### PR TITLE
Added changes for FIPS enable

### PR DESCRIPTION
**Pull request by gauravpbankar**
Signed-off-by: gauravpbankar <gauravpbankar@gmail.com>

### CODE REVIEW

**Review by Prajyot-Parab — DISMISSED** [review]
/lgtm

File: playbooks/roles/ocp-config/templates/install-config.yaml.j2 [hunk 1]
```diff
@@ -22,6 +22,7 @@ networking:
   - {{ service_network }}
 platform:
   none: {}
+fips: {{ fips_compliant }}
```
Comment (yussufsh) [inline playbooks/roles/ocp-config/templates/install-config.yaml.j2:25]: What is `fips_compliant`? 
Please test your code.

**Review by yussufsh — CHANGES_REQUESTED** [review]

File: playbooks/roles/ocp-config/templates/install-config.yaml.j2 (hunk 1 above)
Comment (rpsene) [reply playbooks/roles/ocp-config/templates/install-config.yaml.j2:25]: @yussufsh FIPS https://docs.openshift.com/container-platform/4.7/installing/installing-fips.html
This is must have feature for OCP which needs to be set during installation.

Comment (yussufsh) [reply playbooks/roles/ocp-config/templates/install-config.yaml.j2:25]: I have no doubt that install-config.yaml does support fips.
My concern is with Ansible variable naming here. `fips_compliant` is read here and its `fips` in defaults, examples, README.

**Comment by gauravpbankar** [comment]
git history incorrect due to improper squash so creating new fresh PR request

**Pull request closed without merging**